src/flutter/shell/platform/linux_embedded/surface/environment_egl_drm_eglstream.h

```diff
@@ -0,0 +1,119 @@
+// Copyright 2021 Sony Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_LINUX_EMBEDDED_SURFACE_ENVIRONMENT_EGL_DRM_EGLSTREAM_H_
+#define FLUTTER_SHELL_PLATFORM_LINUX_EMBEDDED_SURFACE_ENVIRONMENT_EGL_DRM_EGLSTREAM_H_
+
+#include <EGL/egl.h>
+#include <EGL/eglext.h>
+
+#include <cstring>
+
+#include "flutter/shell/platform/linux_embedded/logger.h"
+#include "flutter/shell/platform/linux_embedded/surface/environment_egl.h"
+
+namespace flutter {
+
+class EnvironmentEglDrmEglstream : public EnvironmentEgl<void> {
+ public:
+  EnvironmentEglDrmEglstream(int drm_device) : EnvironmentEgl<void>() {
+    if (!SetEglExtensionFunctionPointers()) {
+      LINUXES_LOG(ERROR) << "Failed to set extension function pointers";
+      return;
+    }
+    auto device = GetEglDevice();
+    if (device == EGL_NO_DEVICE_EXT) {
+      LINUXES_LOG(ERROR) << "Couldn't find EGL device";
+      return;
+    }
+
+    EGLint attribs[] = {EGL_DRM_MASTER_FD_EXT, drm_device, EGL_NONE};
+    this->display_ =
```

Comment:
Why do you use `this`?

Reply:
Sorry, no need to use this. I removed it.